pkg/trainer/replicas.go

```diff
@@ -88,7 +94,67 @@ func (s *TFReplicaSet) Labels() KubernetesLabels {
 		"runtime_id": s.Job.job.Spec.RuntimeId})
 }
 
-func (s *TFReplicaSet) Create() error {
+// Transforms the tfconfig to work with grpc_tensorflow_server
+func transformClusterSpecForDefaultPS(clusterSpec ClusterSpec) string {
+
+	// sort by keys to make unit testing easier
+	keys := []string{}
+	for k := range clusterSpec {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	var buf bytes.Buffer
+	isFirstJob := true
+	for _, k := range keys {
```

Comment:
nit: I'd prefer this code to be written as a series of joins so we don't have to have statements dealing with first character

so something like (without the syntax errors)

```
jobs = []string{}
for _, jobType := range keys {
   hosts := []string{}
  for _, h := range clusterSpec[jobType] {
     hosts = append(hosts, h)
   }
   s = jobType + "|" + strings.join(hosts, ";")
  jobs = append(jobs, s)
}
return strings.join(jobs, ",")
```
The code might not generate the correct result but hopefully it illustrates what I mean by using strings.join